src/modules/claim/claim.service.spec.ts

```diff
@@ -1,15 +1,144 @@
 import { Test, TestingModule } from '@nestjs/testing';
-import { ClaimService } from './services';
-
+import { ClaimService, ClaimVerificationService } from './services';
+import { Logger } from '../logger/logger.service';
+import { RoleService } from '../role/role.service';
+import { TypeOrmModule, TypeOrmModuleOptions } from '@nestjs/typeorm';
+import { Claim } from './entities/claim.entity';
+import * as TestDbCOnfig from '../../../e2e/config';
+import { AssetsService } from '../assets/assets.service';
+import { Connection, EntityManager, QueryRunner } from 'typeorm';
+import { RoleClaim } from './entities/roleClaim.entity';
+import { ConfigService } from '@nestjs/config';
+import { ethrReg } from '@ew-did-registry/did-ethr-resolver';
+import { Methods } from '@ew-did-registry/did';
+import { EthereumDIDRegistry } from '../../ethers/EthereumDIDRegistry';
+import { deployContract, MockProvider } from 'ethereum-waffle';
+import { Provider } from '../../common/provider';
+import { Asset, AssetsHistory } from '../assets/assets.entity';
+import { DIDService } from '../did/did.service';
+import { EventEmitter2 } from '@nestjs/event-emitter';
+import { SchedulerRegistry } from '@nestjs/schedule';
 // TODO: fix test so pending can be removed
-xdescribe('ClaimService', () => {
-  let service: ClaimService;
+// private readonly roleService: RoleService,
+// private readonly logger: Logger,
+// @InjectRepository(RoleClaim)
+// private readonly roleClaimRepository: Repository<RoleClaim>,
+// @InjectRepository(Claim)
+// private readonly claimRepository: Repository<Claim>,
+// private readonly assetService: AssetsService,
+// private claimVerificationService: ClaimVerificationService
+// ) {
+// this.logger.setContext(ClaimService.name);
+// }
+const MockLogger = {
+  log: jest.fn(),
+  error: jest.fn(),
+  setContext: jest.fn(),
+  info: jest.fn(),
+  debug: jest.fn(),
+};
+
+const MockClaimVerificationService = {
+  log: jest.fn(),
+  error: jest.fn(),
+  setContext: jest.fn(),
+  info: jest.fn(),
+  debug: jest.fn(),
+};
+
+//let service: DIDService;
+const provider = new MockProvider();
+let didRegistry: EthereumDIDRegistry;
+const cachedIdentity = provider.getWallets()[0];
 
+const MockConfigService = {
+  get: jest.fn((key: string) => {
+    if (key === 'DID_SYNC_ENABLED') {
+      return 'false';
+    } else if (key === 'DID_REGISTRY_ADDRESS') {
+      return didRegistry.address;
```

Comment:
@JGiter I was planning on adding tests to the claim.service test file, if this sounds okay with you.